readme.md

```diff
@@ -193,6 +193,8 @@ export default tw;
 import tw from './lib/tailwind';
 ```
 
+> ⚠️ Make sure to use `module.exports = {}` instead of `export default {}` in your `tailwind.config.ts` file, as the latter is not supported.
```

Comment:
did you mean to write `.ts`? Shouldn't it be `.js`? Can you write a tailwind config in typescript?  Either way, the same problem manifests with a `.js` extension.

Reply:
I wrote `.ts` as I am using the `.ts` one, but indeed it also appears with the `.js` one. 

Should I make it "... in your `tailwind.config.ts` or `tailwind.config.js` file ..."?

Reply:
Yep, done!